index.html

```diff
@@ -5,11 +5,56 @@
     <title>Task2-1</title>
   </head>
   <body>
-
-
+      <h1 class="main-title">FizzBuzz問題</h1>
+      <p>
+        FizzNum：
+        <input type="text" name="number" id="fizz" placeholder="整数値を入れてください">
+      </p>
+      <p>
+          BuzzNum：
+        <input type="text" name="number" id="buzz" placeholder="整数値を入れてください">
+      </p>
+      <button type="button" name="name" value="value" id="btn">実行</button>
+      <div id="output">
+        <p>[出力]</p>
+      </div>
     <script>
       'use strict';
-      
+      const btn = document.getElementById('btn');
+      btn.addEventListener('click', () => {
+        const FizzNumber = Number(document.getElementById('fizz').value);
+        const BuzzNumber = Number(document.getElementById('buzz').value);
+        const outputArea = document.getElementById('output');
+        outputArea.innerHTML = '';
+        const tag = document.createElement('p');
+        tag.textContent = '[出力]';
+        outputArea.appendChild(tag);
+        if(!Number.isInteger(FizzNumber) || !Number.isInteger(BuzzNumber) || FizzNumber === 0 || BuzzNumber === 0 || FizzNumber == "" || BuzzNumber == ""){
+          let errorTag = '整数値を入れてください';
+          const ptag = document.createElement('p');
+          ptag.textContent = errorTag;
+          const ptagArea = document.getElementById('output')
+          outputArea.appendChild(ptag);
+          return;
+        }
+        for (let i = 1; i <= 99; i++) {
+          let value = '';
+          let F = `Fizz`;
+          let B = `Buzz`;
```

Comment:
- 変数名はわかりやすい名前にしましょう
- 再代入はしないと思いますので `const` にしましょう

```suggestion
          const fizzString = `Fizz`;
          const buzzString = `Buzz`;
```